compiler/shared_ast/interpreter.mli

```diff
@@ -59,6 +61,12 @@ val interpret_program_lcalc :
     providing for each argument a thunked empty default. Returns a list of all
     the computed values for the scope variables of the executed scope. *)
 
+val addcustom :
+  (('a, 'b, 'c) interpr_kind, 't) gexpr ->
+  (('a, 'b, yes) interpr_kind, 't) gexpr
+(** Typing shenanigan to add custom terms to the AST type. This is an identity
```

Comment:
This comment is good for the ml file, but the mli doc should just state that it's a type conversion